app.js

```diff
@@ -0,0 +1,15 @@
+import fs from 'node:fs';
+
+import { info } from './os-module.js';
+let data = info();
+const { arch, type, cpus, freemem, totalmem } = data;
+const arr = [arch, type, cpus, freemem, totalmem];
+
+arr.forEach((item) => {
+  fs.appendFile('info.txt', item, (err) => {
```

Comment:
Since all the data is available, it is better to use the asynchronous writeFile method instead of appendFile to save everything at once without a forEarch loop.
Doc: https://nodejs.org/en/learn/manipulating-files/writing-files-with-nodejs